### PR TITLE
Refine features

### PR DESCRIPTION
**Pull request by Stephen-Gates**
update features

- version 2 features
- some v1 corrected to address bugs
- rewrite keyboard shortcuts help
- tweak screen sizes
- refine About
- refine open licences

### CODE REVIEW

File: src/main/menu.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ exports.menu = [
       }, {
         type: 'separator'
       }, {
+// hide below in Windows and Linux
```
Comment (Stephen-Gates) [inline src/main/menu.js]: Note I've identified where platform specifics are required

File: src/main/menu.js [hunk 2]
```diff
@@ -159,94 +155,31 @@ exports.menu = [
...
   }, {
     label: 'Edit',
     submenu: [
       {
-        label: 'Undo',
-        accelerator: 'CmdOrCtrl+Z',
-        click: function() {
-          BrowserWindow.getFocusedWindow().webContents.send('editUndo')
-        }
+        role: 'undo'
```
Comment (Stephen-Gates) [inline src/main/menu.js]: using simpler role: definition. Behaviour appears to be the same.

File: src/main/menu.js [hunk 3]
```diff
@@ -287,6 +220,24 @@ exports.menu = [
       }
     ]
   }, {
+    label: 'View',
```
Comment (Stephen-Gates) [inline src/main/menu.js]: Not sure if we'll keep this top level menu - explore with sponsor

File: src/main/menu.js [hunk 4]
```diff
@@ -374,54 +302,68 @@ exports.menu = [
...
+        }
+      }, {
+        label: 'Publish Data Package to',
         enabled: false,
         submenu: [
           {
             label: 'CKAN',
             enabled: false
-            //          , icon: '/static/img/locked.svg'
+//          , icon: '/static/img/locked.svg'
```
Comment (Stephen-Gates) [inline src/main/menu.js]: A failed attempt at inserting locked icons into premium features

File: src/renderer/components/DataPackage.vue [hunk 5]
```diff
@@ -14,21 +14,27 @@
         <label for="description">Dataset description</label>
         <textarea class="form-control" id="description" name="description" rows="3"></textarea>
       </div>
+<!-- this would be an ideal place to link to in-app help -->
       <div class="form-group">
         <label for="license">License</label>
+<!-- to be sourced from http://opendefinition.org/licenses/#recommended-conformant-licenses plus other selected open licenses -->
```
Comment (Stephen-Gates) [inline src/renderer/components/DataPackage.vue]: This code may need to move Partial component

File: src/renderer/components/DataPackage.vue [hunk 6]
```diff
@@ -14,21 +14,27 @@
...
-          <option value="ogl-uk">UK Open Government Licence</option>
-          <option value="odc-by">Open Data Commons Attribution License</option>
+          <option value="cc-by">Creative Commons Attribution 4.0 (CC-BY-4.0)</option>
+          <option value="cc-by-sa">Creative Commons Attribution Share-Alike 4.0 (CC-BY-SA-4.0)</option>
+          <option value="cc-zero">Creative Commons CCZero (CC0)</option>
+          <option value="pdm">Public Domain</option>
+          <option value="odc-by">Open Data Commons Attribution License (ODC-BY)</option>
           <option value="odc-odbl">Open Data Commons Open Database License (ODbL)</option>
           <option value="odc-pddl">Open Data Commons Public Domain Dedication and Licence (PDDL)</option>
+          <option disabled role=separator> ─── </option>
```
Comment (Stephen-Gates) [inline src/renderer/components/DataPackage.vue]: Not sure if this is the best way to separate Country specific licences

File: src/renderer/components/Home.vue [hunk 7]
```diff
@@ -4,7 +4,7 @@
     <nav class="navbar navbar-default">
       <div class="container-fluid">
         <div class="navbar-header">
-          <a class="navbar-brand">Data Curator</a>
+          <a class="navbar-brand"></a>
```
Comment (Stephen-Gates) [inline src/renderer/components/Home.vue]: partially addresses OCC-207

File: src/renderer/partials/About.vue [hunk 8]
```diff
@@ -41,21 +43,11 @@ export default {
...
-          }
-        },
-        {
-          label: 'Create usable open data'
+          image: '/static/img/data-curator-banner.png',
+          link: 'https://github.com/ODIQueensland/data-curator/tree/master/'
         },
         {
-          label: '0.1.0'
+          label: 'version: 0.2.2'
```
Comment (Stephen-Gates) [inline src/renderer/partials/About.vue]: can this be automated?

File: static/css/keyboard-help.styl [hunk 9]
```diff
@@ -0,0 +1,11 @@
+/*stylus and vue loaders cannot see the ids/classes not in the vue page*/
+
+#body-panel
+  background-position center
+  opacity 0.8
+
+.panel, .container-fluid
+  background transparent !important
+
+/* add styling for vertical-align:top, banded table rows, th styling, table width, a, a.hover, h1, h2, h3 */
```
Comment (Stephen-Gates) [inline static/css/keyboard-help.styl]: Help requested

File: src/renderer/components/KeyboardHelp.vue [hunk 10]
```diff
@@ -1,121 +1,454 @@
 <template>
   <div id="container" class="container-fluid">
-  <h1>Editor Keyboard Shortcuts</h1>
-  <h2>Navigation</h2>
+
+  <h1>Keyboard Shortcuts</h1>
+
+<!-- consider index but how to do internal links?
```
Comment (Stephen-Gates) [inline src/renderer/components/KeyboardHelp.vue]: Help requested

**Comment by Stephen-Gates** [comment]
@mattRedBox ready for review and merge if OK

File: README.md [hunk 11]
```diff
@@ -176,12 +176,16 @@ We're considering:
 
 ### Building a new package
 
+To build Data Curator for macOS locally:
+
+`yarn build`
```
Comment (Stephen-Gates) [inline README.md]: `yarn build` needs something else as it losses toolsbar and about images in the build process

**Comment by Stephen-Gates** [comment]
@mattRedBox updated menu as agreed with Sponsor agreed on 4/9/17. Search for TO DO: where I need you help to insert Platform switches before merging

File: src/main/menu.js [hunk 12]
```diff
@@ -32,23 +31,26 @@ for (var format in file_formats) {
...
+//   enabled: false,
+//   click: function() {
+//     fileActions.saveAsCustom()
+//   }
+// })
 
 exports.menu = [
   {
+// Only show the Data Curator menu for macOS.
+// TO DO: add test for macOS...
```
Comment (Stephen-Gates) [inline src/main/menu.js]: @mattRedBox Add platform test here

File: src/main/menu.js [hunk 13]
```diff
@@ -347,112 +314,109 @@ exports.menu = [
...
+//          , icon: '/static/img/locked.svg'
+//          }
+//        ]
+//      }
     ]
   }, {
+// TO DO: Update this menu if Data Curator is a multi window app    
     label: 'Window',
     submenu: [
+// TO DO: hide below in Windows and Linux
```
Comment (Stephen-Gates) [inline src/main/menu.js]: Add Platform test

File: src/main/menu.js [hunk 14]
```diff
@@ -347,112 +314,109 @@ exports.menu = [
...
       }, {
         label: 'Next Tab',
-        enabled: false,
         accelerator: 'CmdOrCtrl+Right'
       }, {
         label: 'Previous Tab',
-        enabled: false,
         accelerator: 'CmdOrCtrl+Left'
       }, {
+// TO DO: hide below in Windows and Linux
```
Comment (Stephen-Gates) [inline src/main/menu.js]: Add Platform test

File: src/renderer/components/Home.vue (hunk 7 above)
Comment (Stephen-Gates) [reply src/renderer/components/Home.vue]: May add back later now the Data Curator menu is removed for Windows. Will decide in next release 3..5

File: test/features/.nav [hunk 15]
```diff
@@ -10,14 +10,16 @@
   - open-semi-colon-separated-file.feature
```
Comment (Stephen-Gates) [inline test/features/.nav]: @Stephen-Gates Will need to update this next time to reflect recent menu changes

File: README.md (hunk 11 above)
Comment (mattRedBox) [reply README.md]: @Stephen-Gates 
Should be `yarn run build` - no images should be lost now with path changes (let me know if still not the case)
Yes just need to bump version number in package.json - about panel will read from this.

Comment (Stephen-Gates) [reply README.md]: Yep that worked fine @mattRedBox

File: src/renderer/partials/About.vue (hunk 8 above)
Comment (mattRedBox) [reply src/renderer/partials/About.vue]: The version is automated: will put the image and link in new about partial.

**Pull request merged**